script.js

```diff
@@ -7,6 +7,136 @@ const menuLine1 = document.querySelector('.line-1');
+    popupTittle: 'Multi-Post Stories<br/>Gain+Glory4',
+    tittle: '<h2 class="cardtitle">Multi-Post Stories<br/>Gain+Glory4</h2>',
+    popupImage: '<img src="./Images/Snapshoot-Portfolio.svg" alt="popupimg"/>',
+    technologies:
+      '<ul class="proglist"><li class="prg1">Ruby on rails</li><li class="prg2">css</li><li class="prg3">JavScript</li><li class="prg4">html</li></ul>',
+    button:
+      '<button type="button" id="cardbutton3" class="cardbutton" onclick="clickedButton(this.id)">See Project</button>',
+    description: 'This is a great project I did for a company 4',
+    buttonLive:
+      '<button class="see-live-button"><p>See Live</p><i><img src="./Images/Icon-live.svg" alt=""></i></button>',
+    buttonSource:
+      '<button class="see-source-button"><p>See Source</p><i><img src="./Images/See-Source-Icon.svg" alt=""></i></button>',
+  },
+];
+
+const cardElementsArray4 = [
+  {
+    img: '<div class="cards card4"><div class="cardimg cardimg5"><img class="cardimgs"src="./Images/transparent-img.png"alt="Card img-5"/></div>',
+    popupTittle: 'Multi-Post Stories<br/>Gain+Glory5',
+    tittle: '<h2 class="cardtitle">Multi-Post Stories<br/>Gain+Glory5</h2>',
+    popupImage: '<img src="./Images/Snapshoot-Portfolio.svg" alt="popupimg"/>',
+    technologies:
+      '<ul class="proglist"><li class="prg1">Ruby on rails</li><li class="prg2">css</li><li class="prg3">JavScript</li><li class="prg4">html</li></ul>',
+    button:
+      '<button type="button" id="cardbutton4" class="cardbutton" onclick="clickedButton(this.id)">See Project</button>',
+    description: 'This is a great project I did for a company 5',
+    buttonLive:
+      '<button class="see-live-button"><p>See Live</p><i><img src="./Images/Icon-live.svg" alt=""></i></button>',
+    buttonSource:
+      '<button class="see-source-button"><p>See Source</p><i><img src="./Images/See-Source-Icon.svg" alt=""></i></button>',
+  },
+];
+
+const cardElementsArray5 = [
+  {
+    img: '<div class="cards card5"><div class="cardimg cardimg6"><img class="cardimgs"src="./Images/transparent-img.png"alt="Card img-6"/></div>',
+    popupTittle: 'Multi-Post Stories<br/>Gain+Glory6',
+    tittle: '<h2 class="cardtitle">Multi-Post Stories<br/>Gain+Glory6</h2>',
+    popupImage: '<img src="./Images/Snapshoot-Portfolio.svg" alt="popupimg"/>',
+    technologies:
+      '<ul class="proglist"><li class="prg1">Ruby on rails</li><li class="prg2">css</li><li class="prg3">JavScript</li><li class="prg4">html</li></ul>',
+    button:
+      '<button type="button" id="cardbutton5" class="cardbutton" onclick="clickedButton(this.id)">See Project</button>',
+    description: 'This is a great project I did for a company 6',
+    buttonLive:
+      '<button class="see-live-button"><p>See Live</p><i><img src="./Images/Icon-live.svg" alt=""></i></button>',
+    buttonSource:
+      '<button class="see-source-button"><p>See Source</p><i><img src="./Images/See-Source-Icon.svg" alt=""></i></button>',
+  },
+];
```

Comment:
You have done a great job here. Your project Is almost there but you need to follow the design as much as possible. 

- [ ] There is an unnecessary scroll down when I click on the "See Project" button. 

<img width="669" alt="Capture" src="https://user-images.githubusercontent.com/58504655/127436908-6ea5fe01-7110-40be-aaca-81e721904cb6.PNG">

This is how the original design looks like: 

![Uploading Capture.PNG…]()

Reply:
Perfect, thank you for your review arslanbisharat :smile: , the unnecessary scroll was removed in the desktop version, in the mobile version is still there because all the info does not fit in the normal screen mobile height, but the design is like the Figma template.